posts/dask-summit-takeaway.md

```diff
@@ -0,0 +1,120 @@
+---
+author: Max Grover
+date: 2021-5-21
+tags: dask, conference, open-science
+---
+
+# Dask Distributed Summit 2021 Takeaways
+
+## [Pangeo Session](https://zoom.us/rec/play/bVuz_sZOnzsH8fJdrsvH7T_XgXBg64cRSgbV4PERZByVgoLzUDXXoc1pq02Hcfg1oxE7uzAa919Jly1p.tmy6yMikcfgRVHi2?startTime=1621447260000&_x_zm_rtaid=P6Q_HcGvQa-lvvjzHzq4Rg.1621536889692.ea182bc019aed9fbea835c35b62d5ca5&_x_zm_rhtaid=831)
+
+- Cloud optimized datasets help improve speed of analysis
+  - Entire 4 TB datasets open up in a few seconds
+  - Important to chunk you data - store in object-store
+- Reading netcdf data remotely, especially on the cloud, can take a while...
+  - Even with using dask-delayed, quite slow
+- Open-source projects, such as XGCM, make answering big science questions easier
+  - Ex. looking at oxygen minimum zones, converting to density coords
+- Persistent pain points
+  - Using dask vs. understanding dask - can we make this transition easier for non-daskers?
+  - It can tough to pin down specific parts of code which is causing workers to die
+- We should get more science workflows in the cloud
+  - Ensure these workflows are documented, especially in relation to dask
+- Minimum reproducible examples don't always work - instead, curate domain-specific examples
+- STAC (spatio-temporal asset catalog) can be helpful when working with satellite imagery, making it easy to search for specific datasets
+  - Similar to intake, able to search spatially as well
```

Comment:
Link to stackstac : goal is really fast loading of stac catalogs. See Tom Augspurger's talk that used this.